src/styles/styles.js

```diff
@@ -1466,6 +1466,11 @@ const styles = {
         borderColor: themeColors.borderFocus,
     },
 
+    textInputComposeBorder: {
```

Comment:
NAB: Can you please move this to be below `textInputComposeSpacing` just for visibility.